ExactlyOneSuccessMOP(Object c) {

	Currently conflicts with NestedCommand

       event command after(Object c) :  call(void MopDoWork.command(Object)) && args(c){}
       event succeed after(Object c) :  call(void MopDoWork.succeed(Object)) && args(c){}
       event failure after(Object c) :  call(void MopDoWork.fail(Object)) && args(c){}

       fsm:
       start [
        command -> sent
       ]
       sent [
	succeed -> succeeded
	failure -> error
       ]
       succeeded [
	succeed -> error
       ]
       error [
       
       ]

       @fail{
               System.err.println("error in ExactlyOneSuccess");
       }
       @error{
               System.err.println("A command failed or succeeded twice");
       }
}
